release/release_logs/fetch_release_logs.py

```diff
@@ -199,13 +199,16 @@ def write_results(log_dir: Path, fetched_results: Dict[str, Any]) -> None:
 
 @click.command()
 @click.argument("version", required=True)
-def main(version: str):
+@click.argument("commit", required=True)
+def main(version: str, commit: str):
     log_dir = Path(__file__).parent.joinpath(version)
-    branch = f"releases/{version}"
 
     bk = get_buildkite_api()
     build_dict_list = bk.builds().list_all_for_pipeline(
-        organization=BUILDKITE_ORGANIZATION, pipeline=BUILDKITE_PIPELINE, branch=branch
+        organization=BUILDKITE_ORGANIZATION,
+        pipeline=BUILDKITE_PIPELINE,
+        branch="master",
```

Comment:
maybe make branch also a configurable arg?

also, I think in big/full releases we will still need to get stats from release branch?